### PR TITLE
Updated tests

### PR DESCRIPTION
**Pull request by jonfu86**

### CODE REVIEW

File: client/src/components/FeaturesDetails/FeaturesDetails.jsx [hunk 1]
```diff
@@ -2,42 +2,36 @@ import React from 'react';
...
+            </div>
+          </li>
+        ))}
+
+      </ul>
+    </div>
+  );
 }
 
 FeaturesDetails.defaultProps = {
```
Comment (karebae) [inline client/src/components/FeaturesDetails/FeaturesDetails.jsx:37]: interesting, you don't need proptypes if you have defaultprops?

File: tests/FeaturesDetails.spec.js [hunk 2]
```diff
@@ -3,12 +3,30 @@ import renderer from 'react-test-renderer';
...
-  test('snapshot renders', () => {
-    const features = [{ heading: 'heading for 1', description: 'description string for feature 1', posX: 60, posY: 62}, { heading: 'heading for 2', description: 'description string for feature 2', posX: 70, posY: 32}, { heading: 'heading for 3', description: 'description string for feature 3', posX: 20, posY: 83}];
-    const header = 'THIS IS THE HEADER';
+  describe('Have props', () => {
+    let component;
 
-    const component = renderer.create(<FeaturesDetails features={features} header={header} />);
-    let tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    beforeEach(() => {
```
Comment (karebae) [inline tests/FeaturesDetails.spec.js:9]: this is cool. good job

**Review by karebae — APPROVED** [review]
looks good jon, you wrote those quickly! keep up the good work

File: tests/App.spec.js [hunk 3]
```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { mount } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 import App from '../client/src/components/App';
 import FeaturesDetails from '../client/src/components/FeaturesDetails/FeaturesDetails';
+import ImageViewer from '../client/src/components/ImageViewer/ImageViewer';
 
 describe('App', () => {
```
Comment (ryanminjaeshin) [inline tests/App.spec.js:8]: Those enzyme tests are so cool. Nice 💯

**Review by ryanminjaeshin — APPROVED** [review]
I really got impressed with your jest and enzyme testing. Especially, with jest snapshot. I will look into your codes when I am working on testing.

**Pull request merged**